.gitignore

```diff
@@ -139,3 +139,13 @@ doc/reference/*
 !/panel/dist/css
 !/panel/dist/css/*.css
 .venv2/
+.npm/*
```

Comment:
The files below are added to the root of the Panel project on binder. So to not show them in VS code git changes I remove them.